react-native-gutenberg-bridge/android/src/main/java/org/wordpress/mobile/WPAndroidGlue/WPAndroidGlueCode.java

```diff
@@ -160,14 +161,29 @@ public void setContent(String postContent) {
             return;
         }
 
+        if (mShouldUpdateContent) {
```

Comment:
I think there's some non obvious reasoning behind the logic here so, let's add a comment about what happens here. WDYT?

Reply:
Good one, will add it, thanks :)

Reply:
Added :)

Reply:
Confirmed - addressed in fa8eaabd72f4cf15b4926dc1d7b3acdf50eb96fa 👍

Reply:
Thanks for adding the comment @marecar3 .

> Content can be set directly to RootView only once (per RootView instance)

I guess that it's a convention at this point not to push the content again via the app props, right? I mean, it's already done twice, one in `onCreateView` via `startReactApplication()` and once here in `setContent` via `setAppProperties()`. My impression is that we don't to do it again via props because gutenberg-mobile's state management would be bootstrapped again and we don't want that. Right? If so, let's revise the comment a bit to make this more clear. Sorry if i'm nitpicking about a comment, it's just that there is this important detail we need to convey to our future selves when we read this comment in the future :)

Reply:
Hey, @hypest I agree that we need to be clear and detail with the comment.
I wrote this comment based on how things are working when you test the implementation. 

> // Content can be set directly to RootView only once (per RootView instance)
// otherwise it should be done through module interface

So if you try to pass different conent multiple times thrugh props, with current implementation, nothing will happen, content will stay the same. 

But be free to leave comment that you think it's more suitable for this case and I will update it in this PR.

Reply:
Yes, let's add the "why/because" to this `can be set directly to RootView only once`. Thanks!

Reply:
@hypest done.